src/test/java/org/adoptopenjdk/modulescanner/MainTest.java

```diff
@@ -0,0 +1,22 @@
+package org.adoptopenjdk.modulescanner;
+
+import static org.junit.jupiter.api.Assertions.assertLinesMatch;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+
+class MainTest {
+
+    @Test
+    void testCsvOutput() throws Exception {
+        var testMavenRepo = Paths.get("src/test/resources/test-maven-repo");
+        var targetOutput = Paths.get("target/actual-modulescanner.csv");
```

Comment:
Is this guaranteed to be cleaned up between runs? Normally I'd use https://docs.oracle.com/javase/10/docs/api/java/io/File.html#createTempFile(java.lang.String,java.lang.String)

Reply:
It is replaced by each run. But using a random and temporay target is better here. 👍